README.md

```diff
@@ -1,23 +1,150 @@
+  * [Request](#request)
+  * [Response](#respone)
+* [API Documentation](#api-documentation)
+* [Contributing](#contributing)
+* [License](#license)
 
-Contributing
-------------
+## Quickstart Example
+
+```php
+$pSDK = new onOfficeSDK();
+$pSDK->setApiVersion('latest');
+
+$parametersReadEstate = [
+	'data' => [
+		'Id',
+		'kaufpreis',
+		'lage',
+	],
+	'listlimit' => 10,
+	'sortby' => [
+		'kaufpreis' => 'ASC',
+		'warmmiete' => 'ASC',
+	],
+	'filter' => [
+		'kaufpreis' => [
+			['op' => '>', 'val' => 300000],
+		],
+		'status' => [
+			['op' => '=', 'val' => 1],
+		],
+	],
+];
+
+$handleReadEstate = $pSDK->callGeneric(onOfficeSDK::ACTION_ID_READ, 'estate', $parametersReadEstate);
+
+$pSDK->sendRequests('put the token here', 'and secret here');
+
+var_export($pSDK->getResponseArray($handleReadEstate));
+```
+
+Checkout the [example](/example.php) to see a possible implementation of
+this client.
+
+## Usage
+
+### Client
+
+The `onOfficeSDK` is responsible for creating HTTP Requests and
+receiving HTTP Responses from the official API
+
+```php
+$pSDK = new onOfficeSDK();
+$pSDK->setApiVersion('latest');
+```
+
+Make sure that the correct API version is used for your client.
+By default this value is set to `latest`.
+
+### Parameters
+
+The parameters are transferred as JSON in the HTTP Request.
+The client uses the official
+[PHP array notation](https://www.php.net/manual/en/book.json.php)
+before transforming the array to JSON.
+
+```php
+$parametersReadEstate = [
+	'data' => [
+		'Id',
+		'kaufpreis',
+		'lage',
+	],
+	'listlimit' => 10,
+	'sortby' => [
+		'kaufpreis' => 'ASC',
+		'warmmiete' => 'ASC',
+	],
+	'filter' => [
+		'kaufpreis' => [
+			['op' => '>', 'val' => 300000],
+		],
+		'status' => [
+			['op' => '=', 'val' => 1],
+		],
+	],
+];
+```
+
+### Request
+
+To create a request to the API a an `ACTION_ID` is needed.
+In the class `onOfficeSDK` are several constants defined,
+that can be used, so there is no need to copy these `ACTION_IDs`. 
+
+To send the request an API token and API secret is needed,
+checkout the [official API documentation](#api-documentation)
+how to get these credentials.
+
+```php
+$handleReadEstate = $pSDK->callGeneric(onOfficeSDK::ACTION_ID_READ, 'estate', $parametersReadEstate);
+
+$pSDK->sendRequests('put the token here', 'and secret here');
+```
+
+The return value of `onOfficeSDK::callGeneric` is used to identify the
+equivalent response value.
+`onOfficeSDK::callGeneric` can be called multiple times before sending
+the request to the API via `onOfficeSDK::sendRequests`-
+
+
+### Repsonse
+
+Use the method `onOfficeSDK::getResponseArray` to fetch the data for the response data.
+To identify the response of the request use the return value `onOfficeSDK::callGeneric`.
+
+```php
+var_export($pSDK->getResponseArray($handleReadEstate));
+```
+
+The response will be an array values.
+
+## API Documentation
+
+The API client is developed for the current version of the official API version.

```

Comment:
Maybe use *latest* instead of *current* as this is also the identifier used by the SDK.